app/controllers/users.js

```diff
@@ -0,0 +1,24 @@
+const errors = require('../errors');
+const logger = require('../logger');
+const usersServices = require('../services/users');
+
+const { createHash } = require('../helpers/bcrypt');
+const {
+  statusMessages: { CREATED, NOT_CREATED }
+} = require('../constants');
+
+exports.createUser = async (req, res, next) => {
```

Comment:
We prefer the controllers to be the clearest and cleanest as is possible, something like this:
```
exports.createUser = (req, res, next) => 
    someService(req.body)
    .then(value => res.send({...}))
    .catch(next)
```
And the logic could be moved  to a service or iterator

Reply:
OK. I agree. ¡the controller is a bridge!